src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,87 @@
+package duke;
+
+import duke.exception.DukeException;
+
+import java.util.*;
```

Comment:
Perhaps a break-down of just the classes that you need, instead of importing all?

Reply:
Got it, thanks!